docs/05-reference/winglang-spec.md

```diff
@@ -69,23 +62,15 @@ import TOCInline from '@theme/TOCInline';
 | ------ | ---------------------------------- |
 | `void` | represents the absence of a type   |
 | `nil`  | represents the absence of a value  |
```

Comment:
Let's remove this as well - `nil` is not a dedicated type that the user can create or specify (nor should they be able to)
```suggestion
```